projects/packages/search/jest.config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+module.exports = {
+	preset: '@automattic/calypso-build',
```

Comment:
Let's avoid adding new calypso-build dependencies. See what I did with Jetpack in #21581.